api-audit/src/main/java/com/capitalone/dashboard/service/AuditServiceImpl.java

```diff
@@ -2,26 +2,7 @@
 
 import com.capitalone.dashboard.ApiSettings;
 import com.capitalone.dashboard.misc.HygieiaException;
-import com.capitalone.dashboard.model.AuditStatus;
```

Comment:
@sashi-kan Please do not reformat entire file. Add your changes to existing format , that way it will just show up your changes. Add your imports to existing instead of import pakage